es-design-system/nuxt.config.js

```diff
@@ -13,20 +13,21 @@ export default {
             { name: 'theme-color', content: '#ffffff' },
         ],
         link: [
-            { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+            // Fonts
             {
                 rel: 'stylesheet',
                 href: 'https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&display=swap',
             },
-            { rel: 'apple-touch-icon', sizes: '180x180', href: '/apple-touch-icon.png' },
-            {
-                rel: 'icon', type: 'image/png', sizes: '32x32', href: '/favicon-32x32.png',
-            },
+            // Icon
             {
-                rel: 'icon', type: 'image/png', sizes: '16x16', href: '/favicon-16x16.png',
+                rel: 'icon', type: 'image/x-icon', href: '/favicon.ico', sizes: 'any',
             },
+            { rel: 'icon', href: '/favicon.svg', type: 'image/svg+xml' },
+            // Apple devices will use apple-touch-icon
+            // if you add the webpage as a shortcut to your home screen on an iPhone or iPad
+            { rel: 'apple-touch-icon', sizes: '180x180', href: '/apple-touch-icon.png' },
+            // Android devices will use this for PWAs
             { rel: 'manifest', href: '/site.webmanifest' },
```

Comment:
We probably want to use https://pwa.nuxtjs.org/ to generate this instead.

Reply:
Per the [evilmartians / Andrey Sitnik recommendation](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs), what I have above should address at least 70% of browsers (certainly all the ones we support).

My concern about pwa's icon generation are:
- the lift and frequency of changes to the icon, I'd guess, are quite small. It is potentially more effort keeping 3rd party dependencies up to date
- the required icons are quite small and manageable without a 3rd party library
- there might be some art-direction involved with apple-touch icons, so a programmatically generated icon might not be optimal
- the recommendation is easily adaptable to `@nuxtjs/pwa` in downstream repos if wanted

Reply:
Overall this looks like a nice cleanup so I'm inclined to merge this, and if we want to make a ticket for later on to explore the Nuxt PWA library we can. Could be worth it to model that for other teams if we (a) want ESDS to be a good example of how to solve various things and (b) if we think it's a good approach overall.

Reply:
I'm still in favor of using pwa instead but we can merge and discuss more later.